docs/conf/eu/2017/cfp.rst

```diff
@@ -83,22 +84,22 @@ an anchor session.
 Speaker benefits & logistics
 ----------------------------
 
+Speaking at Write the Docs is a fantastic way to participate in the community.
+
 If you are selected to speak at Write the Docs, we will waive your attendance
 fee. We regret that we are not able to cover the cost of travel or lodging for
-all speakers at this time. By not covering travel costs for our speakers, we are
-able to keep ticket prices low and to make the conference accessible to the
-largest number of participants possible. If your proposal is accepted but you
-are unable to attend due to travel costs, please let us know and we will do our
-best to help make it happen.
+all speakers at this time.
 
-Speaking at Write the Docs is a fantastic way to participate in the community.
+By not covering travel costs for our speakers, we are able to keep ticket prices
+low and to make the conference accessible to the largest number of participants possible.
+
+If your proposal is accepted but you are unable to attend due to travel costs, please
+let us know and we will do our best to help make it happen.
 
-Finally, you’ll hear from us with our proposal decisions by **15th June**.
+*Please note that for unconference sessions we will not be able to waive your attendance fee*.
+We hope to have many good conversations, but can't afford to offer each Unconference session leader a free ticket.
 
-If your Unconference session is selected,
-we won't be offering a free ticket.
-We hope to have many good conversations,
-but can't afford to offer each Unconference session leader a free ticket.
+You’ll hear from us with our proposal decisions by **15th June**.
 
 Questions?
```

Comment:
Just a thought, but we've already released the CFP -- is it cool to change it after people have already submitted? Should we ping them to tell them?